bin/client_package/config/update-resolv-conf

```diff
@@ -20,7 +17,11 @@ NMSRVRS=()
 SRCHS=()
 adapters=()
 
+# TODO Investigate DNS on Linux environment
 NETWORKSETUP=/usr/sbin/networksetup
+if [ ! -x "$NETWORKSETUP" ]; then
```

Comment:
NETWORKSETUP=(which networksetup)
if [ - "$NETWORKSETUP" ]; then //not OSX
don't rely on hardcoded path - maybe we decide to bundle our own

Reply:
Removed my logic, waiting for your fix